### ISSUE TITLE
problems with new gdb syntax in ubuntu 22

### ISSUE THREAD

**Issue by sci-42ver** [RAttab/dotfiles#2]
small tweaks with `colors.gdb` based on [this](https://sourceware.org/gdb/onlinedocs/gdb/Logging-Output.html)
here in '~/.gdbinit' source `colors.gdb` with `source ~/dotfiles/colors.gdb`
`colors.gdb` has:
```gdb
shell test -e ~/.gdb-color-pipe && rm ~/.gdb-color-pipe
shell touch ~/.gdb-color-pipe && echo " " >> ~/.gdb-color-pipe && cat ~/.gdb-color-pipe
# shell ls ~

define setup-color-pipe
    shell rm -f ~/.gdb-color-pipe
    set logging redirect on
    set logging enabled on  
    set logging file ~/.gdb-color-pipe
end

define cleanup-color-pipe
    set logging enabled off 
    set logging redirect off
    shell rm -f ~/.gdb-color-pipe
end
```

but in gdb
```gdb
>>> bt
cat: /home/czg/.gdb-color-pipe: No such file or directory
```
but if then do task in gdb
```gdb
>>> !touch ~/.gdb-color-pipe
>>> bt
#0  main (argc=1, argv=0x7fffffffd858) at 2.89.c:43
>>> s
>>> s
>>> frame 
cat: /home/czg/.gdb-color-pipe: No such file or directory
>>> bt
#0  __srandom (x=1678962474) at ./stdlib/random.c:209
#1  0x0000555555555529 in init_seed () at ./lib/random.c:9
#2  0x0000555555555337 in main (argc=1, argv=0x7fffffffd858) at 2.89.c:43
```
`bt` works but `frame` fails which is so weird

Is there one method to see detailed what gdb has done like `set -v -x -T` in `bash`?

**Comment by RAttab** [comment]
While I appreciate the attempt to modernize the script, given that the newer versions of gdb have builtin color highlighting, I've stopped using it a long time ago as it was a pretty insane hack to begin with. I'm also aware that gdb now has a python interface which would be much more suited to implementing something like this and would probably remove the need for the brittle logging mechanism in use here.

Short version, I would recommend not going anywhere this script which I only keep around for posterity. In fact, I just noticed that I  have a commit locally which I forgot to push for cleaning up the dead gdb scripts including this one.
